package.json

```diff
@@ -35,7 +35,8 @@
     "@babel/plugin-proposal-export-default-from": "7.5.2",
     "@babel/preset-env": "7.5.5",
     "@babel/preset-react": "7.0.0",
-    "@testing-library/react": "8.0.1",
+    "@testing-library/react": "8.0.9",
+    "@testing-library/react-hooks": "3.2.1",
```

Comment:
I used another testing-library for hooks so we don't need to write mock components to represent the results of hooks any more.

@testing-library/react-hooks used a feature in react@^16.9.
It doesn't affect the hooks themselves but only needs react to be over 16.9 in the dev environment.
I'm not very sure if this is a good way to do it.